src/private/remoteCamera.ts

```diff
@@ -0,0 +1,359 @@
+     */
+    id: string;
+    /**
+     * Reason the  control session was terminated.
+     */
+    terminatedReason?: SessionTerminatedReason;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate the reason the session was terminated.
+   */
+  export enum SessionTerminatedReason {
+    None,
+    ControlDenied,
+    ControlNoResponse,
+    ControlBusy,
+    AckTimeout,
+    ControlTerminated,
+    ControllerTerminated,
+    DataChannelError,
+    ControllerCancelled,
+    ControlDisabled,
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Fetch a list of the participants with controllable-cameras in a meeting.
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * result can either contain the CapableParticipants object, incase of a successful fetch or null when it fails
+   * result: object that contains an array of participants with controllable-cameras
+   */
+  export function getCapableParticipants(callback: (error: SdkError, result: CapableParticipants) => void): void {
+    if (!callback) {
+      throw new Error('[remoteCamera.getCapableParticipants] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('remoteCamera.getCapableParticipants');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Request control of a participant's camera.
+   * @param participant Participant specifies the participant to send the request for camera control.
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * result can either contain the true/false value, incase of a successful request or null when it fails
+   * result: True means request was accepted and false means request was denied
+   */
+  export function requestControl(
+    participant: ParticipantInput,
+    callback: (error: SdkError, result: boolean) => void,
+  ): void {
+    if (!participant) {
+      throw new Error('[remoteCamera.requestControl] Participant cannot be null');
+    }
+    if (!callback) {
+      throw new Error('[remoteCamera.requestControl] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('remoteCamera.requestControl', [participant]);
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Send control command to the participant's camera.
+   * @param ControlCommand ControlCommand specifies the command for controling the camera.
+   * @param callback Callback to invoke when the command response returns.
+   */
+  export function sendControlCommand(ControlCommand: ControlCommand, callback: (error: SdkError) => void): void {
```

Comment:
do we need to specify which participant we are sending Control to or it depend on current selected participant?

Reply:
You can only be controlling one participant's camera at a time for our use case; perhaps if the need arises, it could be added later